src/soundvectors.py

```diff
@@ -1,353 +1,439 @@
-        "velaric": 0, "long": 0, "ant": 0, "distr": 0, "strid": 0, },
-    "raised": {"cons": 0, "syl": 0, "son": 0, "cont": 0, "delrel": 0, "lat": 0, "nas": 0, "voi": 0, "sg": 0, "cg": 0,
-        "pharyngeal": 0, "laryngeal": 0, "cor": 0, "dorsal": 0, "lab": 0, "hi": 1, "lo": 0, "back": 0, "round": 0,
-        "velaric": 0, "long": 0, "ant": 0, "distr": 0, "strid": 0, }, }, "relative_articulation": {
-    "centralized": {"cons": 0, "syl": 0, "son": 0, "cont": 0, "delrel": 0, "lat": 0, "nas": 0, "voi": 0, "sg": 0,
-        "cg": 0, "pharyngeal": 0, "laryngeal": 0, "cor": 0, "dorsal": 0, "lab": 0, "hi": 0, "lo": 0, "back": 0,
-        "round": 0, "velaric": 0, "long": 0, "ant": 0, "distr": 0, "strid": 0, },
-    "mid-centralized": {"cons": 0, "syl": 0, "son": 0, "cont": 0, "delrel": 0, "lat": 0, "nas": 0, "voi": 0, "sg": 0,
-        "cg": 0, "pharyngeal": 0, "laryngeal": 0, "cor": 0, "dorsal": 0, "lab": 0, "hi": 0, "lo": 0, "back": 0,
-        "round": 0, "velaric": 0, "long": 0, "ant": 0, "distr": 0, "strid": 0, },
-    "advanced": {"cons": 0, "syl": 0, "son": 0, "cont": 0, "delrel": 0, "lat": 0, "nas": 0, "voi": 0, "sg": 0, "cg": 0,
-        "pharyngeal": 0, "laryngeal": 0, "cor": 0, "dorsal": 0, "lab": 0, "hi": 0, "lo": 0, "back": 0, "round": 0,
-        "velaric": 0, "long": 0, "ant": 0, "distr": 0, "strid": 0, },
-    "retracted": {"cons": 0, "syl": 0, "son": 0, "cont": 0, "delrel": 0, "lat": 0, "nas": 0, "voi": 0, "sg": 0, "cg": 0,
-        "pharyngeal": 0, "laryngeal": 0, "cor": 0, "dorsal": 0, "lab": 0, "hi": 0, "lo": 0, "back": 0, "round": 0,
-        "velaric": 0, "long": 0, "ant": 0, "distr": 0, "strid": 0, }, },
-    "centrality": {"back": {"back": 1}, "central": {}, "front": {"front": 1}, "near-back": {"back": 1},
-        "near-front": {"front": 1}, }, "rhotacization": {"rhotacized": {}},
-    "height": {"close": {"hi": 1, "tense": 1}, "close-mid": {"tense": 1}, "mid": {}, "near-close": {"hi": 1},
-        "near-open": {"lo": 1}, "open": {"lo": 1, "tense": 1}, "open-mid": {}, },
-    "friction": {"with-friction": {"son": -1}}, "roundedness": {"rounded": {"round": 1}, "unrounded": {"round": -1}},
-    "rounding": {"less-rounded": {"round": 1}, "more-rounded": {"round": 1}},
-    "start": {"from-high": {"hireg": 1, "hitone": 1}, "from-low": {"loreg": 1}, "from-mid": {},
-        "from-mid-high": {"hireg": 1}, "from-mid-low": {"loreg": 1, "hitone": 1}, },
-    "contour": {"contour": {"contour": 1}, "falling": {"falling": 1}, "rising": {"rising": 1}, "flat": {},
-        "short": {}, },
-    "to_roundedness": {"to_rounded": {"secondrounded": 1}, "to_unrounded": {"secondrounded": -1}, }, }
+clts_features = {
+    "type": {
+        "consonant": FeatureBundle.from_positive_and_negative(
+            ['cons'],
+            ['syl', 'son', 'cont', 'delrel', 'lat', 'nas', 'voi', 'sg', 'cg', 'pharyngeal',
+             'laryngeal', 'cor', 'dorsal', 'lab', 'hi', 'lo', 'back', 'front', 'round',
+             'velaric', 'long']),
```

Comment:
This is the `from_positive_and_negative` factory method in action. From my perspective that allows for fairly transparent initialization.